<launch>
  <arg name="mission_file" default="uuv-ex1"/>
  <arg name="entity_tag" default="uuv_entity"/>
  <arg name="entity_id" default="1"/>
  <arg name="max_contacts" default="100"/>
  <arg name="plugin_tags" default="REAL, SIL"/>

  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>
  <node pkg="nodelet" type="nodelet" name="scrimmage_nodelet" args="load scrimmage_ros/Nodelet standalone_nodelet" output="screen">
    <param name="mission_file" value="$(arg mission_file)" type="str"/>
    <param name="entity_tag" value="$(arg entity_tag)" type="str"/>
    <param name="plugin_tags" type="string" value="$(arg plugin_tags)"/>
    <param name="entity_id" value="$(arg entity_id)" type="int"/>
    <param name="max_contacts" value="$(arg max_contacts)" type="int"/>
    <param name="loop_rate_hz" type="double" value="1"/>
  </node>
</launch>
